### PR TITLE
Docs basic examples initial

### PR DESCRIPTION
**Pull request by RobDWaller**
This PR helps complete some of the examples documentation work highlighted in #6844.

It splits the examples into basic and advanced sections and provides three new basic examples.

- Hello World
- Import Export
- Manage Dependencies

If this PR is acceptable further basic examples will be provided and improvements will be made to the advanced examples.

### CODE REVIEW

**Comment by RobDWaller** [comment]
@ry Ok cool, I can rework things so that is the case. 👍

File: docs/toc.json [hunk 1]
```diff
@@ -67,6 +67,21 @@
   },
   "examples": {
     "name": "Examples",
+    "children": {
+      "examples_basic": "Basic",
+      "examples_advanced": "Advanced"
```
Comment (nayeemrmn) [inline docs/toc.json:72]: These don't seem to be children of this chapter. @lucacasonato Will this work?

Comment (nayeemrmn) [reply docs/toc.json:72]: Oh nevermind, this will be addressed with the previous review.

Comment (RobDWaller) [reply docs/toc.json:72]: I'm going to change it back to just examples as @ry requested.

**Pull request merged**